modules/dnn/src/op_inf_engine.cpp

```diff
@@ -448,7 +460,12 @@ void InfEngineBackendNet::initPlugin(InferenceEngine::ICNNNetwork& net)
                 {
                     InferenceEngine::IExtensionPtr extension =
                         InferenceEngine::make_so_pointer<InferenceEngine::IExtension>(libName);
+
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R1)
                     enginePtr->AddExtension(extension, 0);
+#else
+                    getCore().AddExtension(extension, device_name);
+#endif
```

Comment:
There is extra message on [CI](http://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/2475/steps/test_dnn-opencl/logs/stdio):
> [ WARN:0] DNN-IE: Can't load extension plugin

Perhaps `catch(...) {}` below should be rewritten to capture more information.

Reply:
Fixed by replacing `device_name` which can be both `CPU` and `FPGA` to just `"CPU"`.